schema/SchC1.json

```diff
@@ -0,0 +1,1019 @@
+            ],
+            "minimum": -99999999.99,
+            "maximum": 999999999.99,
+            "examples": [
+                95000.00
+            ],
+            "fec_spec": {
+                "COL_SEQ": 22,
+                "FIELD_DESCRIPTION": "D.2 COLLATERAL VALUE/AMOUNT",
+                "TYPE": "AMT-12",
+                "REQUIRED": null,
+                "SAMPLE_DATA": 95000.00,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null
+            }
+        },
+        "perfected_interest": {
+            "title": "D.3 YES/NO  (Perfected Interest?))",
+            "description": "",
+            "type": [
+                "boolean",
+                "null"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 23,
+                "FIELD_DESCRIPTION": "D.3 YES/NO  (Perfected Interest?))",
+                "TYPE": "A/N-1",
+                "REQUIRED": null,
+                "SAMPLE_DATA": "N",
+                "VALUE_REFERENCE": "Y,N",
+                "RULE_REFERENCE": "Edit: Yes/No"
+            }
+        },
+        "future_income": {
+            "title": "E.1 YES/NO  (Future Income)",
+            "description": "",
+            "type": [
+                "boolean",
+                "null"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 24,
+                "FIELD_DESCRIPTION": "E.1 YES/NO  (Future Income)",
+                "TYPE": "A/N-1",
+                "REQUIRED": null,
+                "SAMPLE_DATA": "N",
+                "VALUE_REFERENCE": "Y,N",
+                "RULE_REFERENCE": "Edit: Yes/No"
+            }
+        },
+        "desc_specification_of_the_above": {
+            "title": "E.2  DESC  (Specification of the above)",
+            "description": "",
+            "type": [
+                "string",
+                "null"
+            ],
+            "minLength": 0,
+            "maxLength": 100,
+            "pattern": "^[ -~]{0,100}$",
+            "fec_spec": {
+                "COL_SEQ": 25,
+                "FIELD_DESCRIPTION": "E.2  DESC  (Specification of the above)",
+                "TYPE": "A/N-100",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null
+            }
+        },
+        "estimated_value": {
+            "title": "E.3 ESTIMATED VALUE",
+            "description": "",
+            "type": [
+                "number",
+                "null"
+            ],
+            "minimum": -99999999.99,
+            "maximum": 999999999.99,
+            "fec_spec": {
+                "COL_SEQ": 26,
+                "FIELD_DESCRIPTION": "E.3 ESTIMATED VALUE",
+                "TYPE": "AMT-12",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null
+            }
+        },
+        "date_depository_account_established": {
```

Comment:
In keeping with our variable pattern for most date fields (date_signed and date_of_election look to be the exceptions), suggest we rename this to depository_account_established_date

Reply:
change was made